R/tidyverse.R

```diff
@@ -107,6 +107,10 @@ use_tidy_description <- function() {
 #' @export
 #' @rdname tidyverse
 use_tidy_dependencies <- function() {
+  if (!check_has_package_doc()) {
```

Comment:
Just noting for the sake of creating shared understanding:

I think `check_*()` functions should not have meaningful return values, i.e. they exit if the to-be-checked condition doesn't hold.

Do we agree? I think we are awfully close to that being true already.

Reply:
Yeah, I'll change this function. It confused me in the tests so I should've made the change already.

(I think the only exception to this rule is sometimes an argument checking function might do some light massaging (like `tolower()` or `as.integer()`) so `x <- check_x(x)` is ok)

Reply:
Also re: shared understanding:

In the GitHub areas of usethis, I have a few `challenge_*()` functions for when I want to make sure the user understands they're doing something possibly weird / dangerous, but if they accept the risk, we keep going.